### ISSUE TITLE
Controls no longer work after full screen

### ISSUE THREAD

**Issue by ccgfarmer** [OpenEmu/OpenEmu#2013]
So here's a weird one: 
I entered full screen mode, tried to exit out using Esc key, and (since it didn't work and I didn't know the key command to exit), I Force Quit out of the application.

Now, whenever I try to use a SNES game, the "A" button is on hyper mode in certain games and the arrow keys don't work at all on some games (games where they previously worked).

I've tried to reset the bindings on OpenEmu, tried to assign the keys to other keys, and tried to reset as much as I could, but the games where the controller is messed up are still messed up.

Any suggestions?

**Comment by gabspeck** [comment]
I can confirm this on El Capitan Beta 4 with a build from git (slightly outdated though).

**Comment by clobber** [comment]
@ccgfarmer Cannot replicate the issue, if I am understanding you which I am not sure. There are 3 ways to exit fullscreen: 1) click the fullscreen button on the HUD bar, 2) hit the shortcut key or 3) click the green button on the title bar just like any OS X app. It is mind boggling how you resorted to force quit. 

Anyway, sure would be nice to know which version of OS X you are using.

@gabspeck Confirm what exactly? You should plainly state the issue with steps to replicate it. Also not sure if your issue is even related to this.

**Comment by gabspeck** [comment]
Alright, I've just updated to El Capitan Beta 5, pulled the latest code from git and rebuilt it (without experimental features) from the command line and cleaned up data from previous versions out of my personal folder as instructed on the wiki page.

I normally use a Dualshock 3 controller paired via Bluetooth, but was able to reproduce this with the keyboard as well.

When I start a game in windowed mode, the game controls work. After switching to fullscreen mode, they stop working (even if I go back to windowed mode). 

The save/load state buttons work at all times though (I've mapped those to the DS3 controller).

Tested with SNES and NES games.
